project-icp/bee-pollinator-app
# Replace Layer Control Placeholder Icons

## Overview

Replaces the placeholder image icons on the basemap selector and the CDL layer toggle.

I got the street and satellite thumbnail images by screenshotting the wireframes.

## Demo

<img width="350" alt="screen shot 2016-11-29 at 5 51 34 pm" src="https://cloud.githubusercontent.com/assets/7633670/20732468/d5e46cb0-b65c-11e6-8f22-dd0da01c57ac.png">

![screen shot 2016-11-29 at 5 51 40 pm](https://cloud.githubusercontent.com/assets/7633670/20732399/878d2f52-b65c-11e6-8a07-965cff7f3082.png)

## Testing Instructions
Pull this branch, and bundle the project

Confirm the 
- new icons appear
- the controls toggle correctly
- the changes didn't break any other styling 

Connects #57
